dev/service-registry/client-javadsl/src/main/java/com/lightbend/lagom/internal/javadsl/registry/ServiceRegistry.java

```diff
@@ -44,7 +45,7 @@ default Descriptor descriptor() {
 		return named(SERVICE_NAME).withCalls(
             restCall(Method.PUT, "/services/:id", this::register),
 		    restCall(Method.DELETE, "/services/:id", this::unregister),
-		    restCall(Method.GET, "/services/:id", this::lookup).withResponseSerializer(CustomSerializers.URI),
+		    restCall(Method.GET, "/services/:id?portName", this::lookup).withResponseSerializer(CustomSerializers.URI),
 		    pathCall("/services", this::registeredServices)
         ).withLocatableService(false);
 		// @formatter:on
```

Comment:
The (javadsl) service descriptor for the Service Registry in Dev Mode now supports searching by `portName`